specr.toml

```diff
@@ -0,0 +1,13 @@
+input = "."
```

Comment:
How does this define which files to use? If it's just "all markdown files recursively" then we probably want to move this all into a subdir at some point, so that it doesn't pick up things e.g. from inside minimize.

Reply:
Agreed, I think when minisyntax & minitest will be merged into this repo, it's a good time to add the `spec` and `tooling` subdirectories.